semantic/semantic.fbs

```diff
@@ -1,5 +1,9 @@
 namespace fbsemantic;
 
+table Fresher {
```

Comment:
I guess you need this so when we do inference on Flux programs you know where to start creating new type variables, so you don't collide with the ones already in the stdlib?

Reply:
Yes.